### PR TITLE
Feature/etherspot balances hook

### PR DESCRIPTION
**Pull request by poocart**
Added `useEtherstpoBalances` hook to pull Etherspot balances of native chain assets and ERC20 tokens.

### CODE REVIEW

File: src/hooks/useEtherspotBalances.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      }
+
+      try {
+        const { items } = await sdkForChainId.getAccountBalances({
+          account: sdkForChainId.state.account.address,
+          chainId,
+        });
+        if (shouldUpdate) setBalances(items);
+      } catch (e) {
+        //
```
Comment (IAmKio) [inline src/hooks/useEtherspotBalances.ts:39]: We should log something here, at least a `console.warn` to show developers that there was a problem with the service as opposed to something that they have done

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just one small change but other than that - all good!

File: src/hooks/useEtherspotBalances.ts (hunk 1 above)
Comment (poocart) [reply src/hooks/useEtherspotBalances.ts:39]: added, thanks!

**Review by IAmKio — APPROVED** [review]
LGTM

**Pull request merged**